ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -239,9 +251,6 @@ public void onDockerChange() {
 	}
 
 	public void onRuntimeChange() {
-		LOGGER.log(Level.INFO,
```

Comment:
Please restore these log entries. They are important to understand what is going on in the application. If log volume is a concern (but quite honestly current log volumes are very modest), the log level can be downgraded a bit.